### PR TITLE
Scenes 2.0

### PR DESCRIPTION
**Pull request by nmaqsood91**

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+            "type": "node",
+            "request": "launch",
+            "name": "Launch Program",
+            "skipFiles": [
+                "<node_internals>/**"
+            ],
+            "program": "${workspaceFolder}/bin/www"
+        }
+    ]
+}
```
Comment (hashimhayat) [inline .vscode/launch.json:17]: Can you rm this file, this should go into gitignore

File: routes/Client/scenes.js [hunk 2]
```diff
@@ -11,6 +11,7 @@ router.get("/", async function(req, res, next) {
   const controller = new ScenesController(req);
   controller.processGetScenes().then(
     async result => {
+      console.log(result)
```
Comment (hashimhayat) [inline routes/Client/scenes.js:14]: We can remove redundant `console.log `

File: views/Client/scenes/assets/js/main.js [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+// Responsive Navbar
+function Toogler() {
+  $('.navbar-nav').toggleClass('responsive');
+}
+
+//Closing nav menu after cliking
+var x = document.getElementsByClassName('nav-item');
+$(x).click(function() {
+  $('.navbar-nav').removeClass('responsive');
+});
```
Comment (hashimhayat) [inline views/Client/scenes/assets/js/main.js:62]: This file was committed twice in two places. We usually keep out `css/js/assets` in the assets file. Let's remove these files from the views folder. The view folder should only have` .hbs` files.

**Comment by hashimhayat** [comment]
@nmaqsood91 We keep the assets in the `public/assets` folder in order to keep this separate. The files in the `views` folder should only be `.hbs`.

Ideally, we should only have `.hbs` files in `labocine-express/views/Client/scenes/`.

About the **videos** folder. I don't know where these videos are used, ideally, these videos in the `videos` folder below should be uploaded to our asset manager (cc: @samimahmood).

Screen Shot 2020-05-14 at 10 12 42 PM

File: src/controllers/Client/scenes/ScenesController.js [hunk 4]
```diff
@@ -34,7 +34,7 @@ class ScenesController extends Controller {
             title: "Scenes | Labocine",
             database: {
               highlighted: result.slice(0, 10),
-              featured: result.slice(10, 20),
+              featured: result.slice(10, 19),
```
[the remark this answers is not in the document]
Comment (nmaqsood91) [reply src/controllers/Client/scenes/ScenesController.js:37]: @hashimhayat it is still random, only the number of videos are set to 9, earlier it was returning 10 which do not fit set into the designs.

**Pull request merged**